scripts/text_classification/fasttext_word_ngram.py

```diff
@@ -30,18 +30,37 @@
 import argparse
 import logging
 import math
+import time
 
 from collections import Counter
 import numpy as np
+import multiprocessing as mp
 from mxnet import nd, autograd
 from mxnet.gluon import nn, HybridBlock
 import mxnet as mx
 import mxnet.gluon as gluon
 import gluonnlp.data.batchify as btf
 
 import gluonnlp
+#import evaluation
```

Comment:
Should this be deleted (same as ` #evaluation.add_parameters(parser)` etc. below)?

Reply:
fixed